synonymdev/bitkit-react-native
# chore(wallet): Upgrade Reanimated Library

### Description
- Upgrades `react-native-reanimated` to `3.1.0`.
- Updated `SwipeToConfirm` component.
- Removes all instances of `LayoutManager`.

### Type of change

### Tests

### QA Notes
- Ensure the reanimated components continue to work as expected throughout the app.
- To test the updated `SwipeToConfirm` component, navigate to the "ReviewAndSend" view by pasting a Bitcoin/Lightning address after tapping "Send", select "Max" amount, slide to confirm and select "No". The slider should reset prior to navigating back to the previous view.
